content/posts/Getting-used-to-user-testing-roleplay.md

```diff
@@ -0,0 +1,13 @@
++++
+title = "Getting used to user testing: Roleplay at the UXForum"
+date = "2022-05-09"
+author = "Eriol Fox"
+cover = "UXForum.png"
+description = "Getting used to user testing: Roleplay at the UXForum"
++++
+
+As part of the UXForum 2022, we hosted a session for anyone who wants to practice user testing facilitation (without the pressure!). Our participants got hands-on experience with a practive role play. We prepared test scripts and role descriptions so that participants could try out the process without worrying about the content. Our goal was to help participants gain the confidence to bring user testing into their OSS projects. The only thing missing was more time - it takes time to familiarize with a new character, so we wished to have 15-30 more minutes. Otherwise, participants came back with learnings and new experiences. 
+
+You're welcome to use the slidedeck and scripts with your team. Let us know what you think! Google folder: https://drive.google.com/drive/folders/1wbSHjEhZVqzWMVkxZT_qTSP3RSuJkgxD
```

Comment:
You could make this a hyper link by doing: `[This Google folder](https://drive.google.com/drive/folders/1wbSHjEhZVqzWMVkxZT_qTSP3RSuJkgxD)`

Rather than have the long google folder link!

Reply:
Sounds great! Could you merge this pull and then I'll do another pull request with that change?